lib/order.rb

```diff
@@ -0,0 +1,69 @@
+require 'csv'
+require_relative 'customer'
+
+class Order
+
+  attr_reader :id
+  attr_accessor :products, :fulfillment_status, :customer
+ 
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id.to_i
+    @products = products
+    @fulfillment_status = fulfillment_status
+    @customer = customer
+
+    valid_stats = [:pending, :processing, :paid, :shipped, :complete]
+    
+    unless valid_stats.include? fulfillment_status
+      raise ArgumentError.new("The given fulfillment status is invalid, please try again.")
```

Comment:
Including the invalid status in the error message is helpful.

```suggestion
      raise ArgumentError.new("The given fulfillment status (#{fulfillment_status}) is invalid, please try again.")
```